src/terminal/parser/ft_fuzzer/fuzzing_directed.h

```diff
@@ -21,12 +21,14 @@
 
 #include <vector>
 #include <limits>
-#include <atlcoll.h>
 #include <assert.h>
 #include <strsafe.h>
 
+#pragma warning(disable:4505)

```

Comment:
After removing atlcoll.h it shows `warning C4505: 'fuzz::_fz_wsz_addFormatChar': unreferenced local function has been removed`. And this warning is disabled in atlbase.h, so I disable it here.

Reply:
That's fine.